src/gluonts/mx/block/quantile_output.py

```diff
@@ -142,6 +148,64 @@ def compute_quantile_loss(
 
         return qt_loss
 
+    def compute_quantile_weights(self):
+        """
+        Compute weight of each quantile
+
+        Math:
+        Let quantiles = [q_1, ..., q_n] with quantile estimates [z_1, ..., z_n].
+        Then, approximated CRPS with the linear interpolation is
+
+        .. math::
+            :nowrap:
+
+                    CRPS = sum_{i=1}^{n-1} 0.5 * (q_{i+1} - q_{i}) * (z_{i+1} + z_{i}).
+
+        Reordering w.r.t. e_j for j=1, ..., n, gives
+        .. math::
+            :nowrap:
+
+                CRPS = 0.5 * (q_2 - q_1) z_1
+                + 0.5 * (q_3 - q_1)  z_2
+                + ....
+                + 0.5 * (q_n - q_{n-2}) z_{n-1}
+                + 0.5 * (q_n - q_{n-1}) z_n
+        , where each coefficient of z_j is quantile weight w_j.
```

Comment:
Maybe the initial comma can be removed from this line?

Also, I’m not sure whether there should be an empty line preceding this?

Reply:
removed comma and added empty line.